__tests__/integration/mirador/companion_windows.test.js

```diff
@@ -8,7 +8,8 @@ describe('Companion Windows', () => {
     await expect(page).toClick('button[aria-label="Toggle window sidebar"]');
 
     await page.waitFor(1000);
-    await expect(page).toClick('button[aria-label="Open information companion window"]');
```

Comment:
This panel is no longer hidden, so instead we check for the presence of a selected info panel